src/Symfony/Component/Validator/Tests/Constraints/WhenTest.php

```diff
@@ -118,6 +118,8 @@ public function testAttributes()
      */
     public function testAttributesWithClosure()
     {
+        $this->markTestSkipped('Requires https://github.com/php/php-src/issues/17851 to be fixed');
```

Comment:
@alexandre-daubois and me are keeping an eye on php/php-src#17851 to revert this change once the bug in PHP has been fixed